src/cert.rs

```diff
@@ -307,21 +307,18 @@ impl<'a> CrlDistributionPoint<'a> {
             //   a DistributionPoint MUST NOT consist of only the reasons field; either distributionPoint or
             //   cRLIssuer MUST be present.
             match (result.distribution_point, result.crl_issuer) {
-                (None, None) => return Err(Error::MalformedExtensions),
-                _ => {}
+                (None, None) => Err(Error::MalformedExtensions),
```

Comment:
This brings the two return values closer together, clearly showing that this the final return value.

Reply:
Ahh! I see, thanks. Sorry that I misunderstood the first time :+1: